pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 2.5.0
```

Comment:
Should this be a major change instead of a minor? It seems like the pipeline is making changes to scientific outputs?

Reply:
Does Warp have true semantic versioning?  The probably yes.

Reply:
Ok! We follow basically semantic versioning ++. Described [here](https://broadinstitute.github.io/warp/docs/About_WARP/VersionAndReleasePipelines#versioning-guidelines), but the TL;DR is that we follow software semantic versioning plus we will also do a major version update if the scientific content of the outputs have changed (even if the interfaces aren't breaking). So software and/or scientific breaking changes are major changes.